src/LobbyClient.hs

```diff
@@ -22,30 +22,31 @@ clientMain api = do
   createBootstrapTemplate "Hastings"
   createChangeNickNameDOM api
   createChatDOM api
-  createLobbyDOM api
+  createLobbyDOM api newGameAPI
```

Comment:
Should a newGameAPI be created both here and on row 28.
It would be more consistent to create the GameAPI in Main.hs, just like the LobbyAPI is created and then passed here as a parameter.